src/reducers/settings.js

```diff
@@ -27,6 +27,9 @@ export default (state = DEFAULT_SETTINGS, action) => {
 
 export const sGetSettings = (state) => state.settings
 
+export const sGetSettingsDisplayProperty = (state) =>
```

Comment:
what's the difference between the new `displayProperty` and the existing `displayNameProperty`?
Would be great if this was explained in the PR description.

Reply:
The `displayProperty` is `settings.keyAnalysisDisplayProperty` and is the one passed to the analytics request:
https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-239/analytics.html#webapi_analytics_query_parameters

`displayNameProperty` is something we derive from the above to use in other requests.

See:
https://github.com/dhis2/data-visualizer-app/blob/feat/plugin-wrapper-for-iframe/src/components/UserSettingsProvider.js#L26-L30

Reply:
Before it was doing the same thing but with a double conversion and the naming was confusing as it was storing the derived value in `displayName` and then converting it back to the original one for using it in the analytics request.